website/docs/03_features/03_configuration.mdx

```diff
@@ -0,0 +1,401 @@
+import TerminalWindow from '@site/src/components/TerminalWindow/TerminalWindow'
```

Comment:
this import on a documentation page looks weird to me. What is the purpose?

Reply:
I think this is to use `<TerminalWindow>` tag below. I would double-check if this code works in the documentation page